docs/apm/traces.asciidoc

```diff
@@ -0,0 +1,36 @@
+[[traces]]
+=== Traces
+
+The *Traces* overview displays the entry transaction for all traces in your application.
+If you're using <<distributed-tracing>>, this view is key to finding the critical paths within your application.
```

Comment:
this renders as "Distributed Tracing" in the UI, whereas it uses sentence case elsewhere. I think if you add <<distributed-tracing, distributed tracing>> it will render as sentence case?

Reply:
Ahh good find! I'm going to update the title to be sentence cased and that should fix this.